.github/workflows/lint-emacs.yml

```diff
@@ -29,3 +29,6 @@ jobs:
 
     - name: Lint the project
       run: eldev -dtT -C compile --warnings-as-errors
+
+    - name: Run tests
```

Comment:
If we keep the tests here we should probably rename the workflow to something like "CI".